events/netherlands/balfolk_nl.yaml

```diff
@@ -91,6 +92,23 @@ events:
     social: true
     price: donation
 
+  - name: Fest Noz Hoorn
+    links:
+      - "https://www.balfolk.nl/agenda/fest-noz-hoorn/"
+      - "https://www.facebook.com/events/2864813930451126/"
+    start: "2022-04-17T12:00:00+02:00"
+    end: "2022-04-17T23:30:00+02:00"
+    country: Netherlands
+    city: Hoorn
+    styles: [balfolk]
+    workshop: true
+    social: true
+    bands:
+      - Beat Bouet Trio
+      - Fahrenheit
+      - La Sauterelle
+    price: €5-€15
```

Comment:
This price doesn't match what the balfolk.nl page says.

Reply:
Was a copy paste error, took the format from another event, fixed in new commit

Reply:
Without workshop it is 20 euro's, with workshop and fairtrade it is 45. Sorry for this error!